shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommandType.java

```diff
@@ -22,6 +22,7 @@
  */
 public enum WireCommandType {
     HELLO(-127, WireCommands.Hello::readFrom),
+    FLUSH_SEGMENT(-126, null),
```

Comment:
Let's make this -2

Reply:
@tkaitchuck  , -2 is already in use by PARTIAL_EVENT; so , I have assigned -3 for PARTIAL_EVENT